java/pom.xml

```diff
@@ -70,6 +70,7 @@
         <module>aiservices/openai</module>
 
         <module>semantickernel-api</module>
+        <module>aiservices/google</module>
```

Comment:
think I would move this up to be with the openai one, would also rename from google to gemini

Reply:
Following .NET naming here.